assets/blocks/learner-courses-block/index.js

```diff
@@ -11,19 +11,19 @@ import metadata from './block.json';
 import { LearnerCoursesIcon as icon } from '../../icons';
 
 export default {
-	title: __( 'Learner Courses', 'sensei-lms' ),
+	title: __( 'Student Courses', 'sensei-lms' ),
 	description: __(
-		'Manage what learners see on their dashboard. This block is only displayed to logged in learners.',
+		'Manage what students see on their dashboard. This block is only displayed to logged in students.',
 		'sensei-lms'
 	),
 	keywords: [
-		__( 'Learner Courses', 'sensei-lms' ),
+		__( 'Student Courses', 'sensei-lms' ),
 		__( 'My Courses', 'sensei-lms' ),
 		__( 'Dashboard', 'sensei-lms' ),
 		__( 'Courses', 'sensei-lms' ),
 		__( 'Enrolled', 'sensei-lms' ),
 		__( 'Student', 'sensei-lms' ),
-		__( 'Learner', 'sensei-lms' ),
+		__( 'Student', 'sensei-lms' ),
```

Comment:
Looks like a duplication of the previous line

Reply:
Yeah, I wasn't 100% sure about this, why did we have this before tbh. I will check this

Reply:
Soo, I've decided to leave both here, this is just a keyword so I guess we want this block searchable by both student and learner :D